frontend/src/pages/Report.tsx

```diff
@@ -135,18 +137,19 @@ export const Report = () => {
           days={[today.toISOString().split("T")[0]]}
           title="Recent issues"
         />
-        {recentIssues.map((issue) => {
-          return (
-            <>
-              <Row
-                recentIssue={issue}
-                onCellUpdate={handleCellUpdate}
-                days={[today]}
-                userId={user.user_id}
-              />
-            </>
-          );
-        })}
+        {recentIssues &&
```

Comment:
Is this changes related to the env file or something else?

Reply:
this is validating that there are recent issues to  do the mapping, otherwise it fails.